pkg/controller/clone-controller.go

```diff
@@ -16,21 +16,26 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 
-	. "kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/pkg/common"
 	expectations "kubevirt.io/containerized-data-importer/pkg/expectations"
 )
 
 const (
-	// pvc annotations
+	//AnnCloneRequest sets our expected annotation for a CloneRequest
 	AnnCloneRequest = "k8s.io/CloneRequest"
-	AnnCloneOf      = "k8s.io/CloneOf"
-	// cloner pods annotations
-	AnnCloningCreatedBy   = "cdi.kubevirt.io/storage.cloningCreatedByController"
-	AnnClonePodPhase      = "cdi.kubevirt.io/storage.clone.pod.phase"
-	CloneUniqueID         = "cdi.kubevirt.io/storage.clone.cloneUniqeId"
+	// AnnCloneOf sets our expected annotation to inidcate where a clone originated from
+	AnnCloneOf = "k8s.io/CloneOf"
+	// AnnCloningCreatedBy provides an annotation to indicate a PVC was created by our clone controller
+	AnnCloningCreatedBy = "cdi.kubevirt.io/storage.cloningCreatedByController"
+	// AnnClonePodPhase provides an annotation to indicate the pod phase of the our clone pod
```

Comment:
again, not specific to the cloner. To see the full scope of use try `grep -rn  <Ann-in-question> pkg`

Reply:
ack, update on the way